### PR TITLE
Arguments for itemTemplate option return dom node without jQuery

### PR DESCRIPTION
**Pull request by ovchinnikov**

### CODE REVIEW

**Review by mrjohn0011 — APPROVED** [review]

File: js/ui/grid_core/ui.grid_core.adaptivity.js [hunk 1]
```diff
@@ -79,7 +80,7 @@ var AdaptiveColumnsController = modules.ViewController.inherit({
             var templateOptions = extend({}, cellOptions, { value: cellValue, text: cellText, column: column });
             that._rowsView.renderTemplate($container, column.cellTemplate, templateOptions, !!$container.closest(document).length);
         } else {
-            var container = $container.get(0);
+            container = $container.get(0);
```
Comment (nightskylark) [inline js/ui/grid_core/ui.grid_core.adaptivity.js:83]: It looks like we do not need this row at all

File: testing/tests/DevExpress.ui/collectionWidget.tests.js [hunk 2]
```diff
@@ -159,7 +159,8 @@ QUnit.test("custom render func, returns jquery", function(assert) {
             testProp: 5
         }],
         itemTemplate: function(item, index, itemElement) {
-            itemElement.append($("<span />").html("Text is: " + String(item.testProp) + ";"));
+            assert.equal(isRenderer(itemElement), config().useJQueryRenderer, "itemElemenet is correct");
```
Comment (nightskylark) [inline testing/tests/DevExpress.ui/collectionWidget.tests.js:162]: Why don't you check if the itemElement is the instance of jQuery? It should be more suitable for this case.

**Review by nightskylark — CHANGES_REQUESTED** [review]

File: js/ui/grid_core/ui.grid_core.adaptivity.js (hunk 1 above)
Comment (ovchinnikov) [reply js/ui/grid_core/ui.grid_core.adaptivity.js:83]: We need this row. <b>container</b> variable can be jQuery object and jQuery doesn't have textContent property...

Comment (nightskylark) [reply js/ui/grid_core/ui.grid_core.adaptivity.js:83]: ok

File: testing/tests/DevExpress.ui/collectionWidget.tests.js (hunk 2 above)
Comment (ovchinnikov) [reply testing/tests/DevExpress.ui/collectionWidget.tests.js:162]: isRenderer method checks if the itemElement is the instance of jQuery.

Comment (nightskylark) [reply testing/tests/DevExpress.ui/collectionWidget.tests.js:162]: I know, but `instanceof jQuery` looks more obvious. Isn't it?

Comment (ovchinnikov) [reply testing/tests/DevExpress.ui/collectionWidget.tests.js:162]: Yes. But this condition doesn't check if itemElement is dom node. We have to be sure that itemElement isn't DxRenderer also.

DxRenderer doesn't instance of jQuery. So this test will pass without jQuery...

Comment (nightskylark) [reply testing/tests/DevExpress.ui/collectionWidget.tests.js:162]: Now I got it, thanks.

**Review by nightskylark — APPROVED** [review]

**Review by jsdmitry — APPROVED** [review]

**Review by dxvladislavvolkov — APPROVED** [review]

**Review by lukyanovas — APPROVED** [review]

**Pull request merged**